packages/components/psammead-most-read/src/testHelpers/index.jsx

```diff
@@ -18,6 +18,18 @@ const lastUpdated = ({ script, service }) => (
   </Timestamp>
 );
 
+export const getServiceVariant = ({ service, variant = '' }) => {
+  // non variant services have a default value
+  const variantOverride =
+    variant === 'default'
+      ? ''
+      : variant.charAt(0).toUpperCase() + variant.substring(1);
+  // ukchina is camel cased in TEXT_VARIANTS
+  const serviceOverride = service === 'ukchina' ? 'ukChina' : service;
```

Comment:
i'm going update storybook-helper to remove the camel casing for ukchina

Reply:
https://github.com/bbc/psammead/pull/2940
this has been merged now, the root directory will just need a bump for it to work on latest storybook